### PR TITLE
Prepare initial release

### PR DESCRIPTION
**Pull request by JanPietrzyk**

### CODE REVIEW

**Review by teiling88 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,108 @@
...
+
+```
+
+## Usage
+
+You always need a configuration that sets up the basic column names of your implementation:
+
+```php
+
+use Shopware\DbalNestedSet\NestedSetConventionsConfig;
```
Comment (bcremer) [inline README.md]: `NestedSetConventionsConfig` was renamend to `NestedSetConfig`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,108 @@
...
+use Shopware\DbalNestedSet\NestedSetFactory;
+use Doctrine\DBAL\Connection;
+
+$writer = NestedSetFactory::createWriter($dbalConnection, $config);
+
+```
+
+### Modify the tree
+
+The library provides a `NestedSetWriter` class that contains all insert, move and update operations. All operations should be reminiscend of `Doctrine\DBAL\Connection::insert()` and `Doctrine\DBAL\Connection::update()` and just require plain data.
```
Comment (bcremer) [inline README.md]: Typo: s/reminiscend/reminiscent/

File: README.md (hunk 1 above)
Comment (JanPietrzyk) [reply README.md]: @bcremer thanks, but this is still work in progress...

File: README.md (hunk 2 above)
Comment (JanPietrzyk) [reply README.md]: @bcremer thanks, but this is still work in progress...

**Pull request merged**